hclee/process_all_in_one.py

```diff
@@ -0,0 +1,349 @@
+                shutil.copy(dcamPath,saveName)
+                print(saveName)
+        else:
+            #Distance Display csv 처리 (모니터, 차량의 경우 Depth 센서가 존재함)
+            ddispPathlist = glob.glob(f"{device}/DistDisp2Face/*.txt")
+            ddispPathlist.sort()
+            for ddispPath in ddispPathlist:
+                frameCountKey = ddispPath.replace("/DistDisp2Face/", "/RGB/").replace("_ddisp_","_rgb_").replace(".txt",".mp4")
+                frameCount = int(frameCountDict[frameCountKey])
+                if os.path.getsize(ddispPath):
+                    with open(ddispPath) as ddispTxt:
+                        ddispValue = ddispTxt.read().splitlines()
+                        ddispValue = ddispValue[:frameCount]
+                    ddispCsv = pd.DataFrame(ddispValue, columns=['dintance'])
+                    saveName = ddispPath.replace(".txt",".csv")
+                    ddispCsv.to_csv(saveName, index=False)
+                    print(saveName)        
+
+        #CamAngle csv 처리
+        camAnglePathList = glob.glob(f"{device}/CamAngle/*.txt")
+        camAnglePathList.sort()
+        for camAnglePath in camAnglePathList:
+            if os.path.getsize(camAnglePath) > 0:
+                with open(camAnglePath) as camAngleTxt:
+                    camAngleValue = camAngleTxt.read().splitlines()
+                    while "" in camAngleValue:
+                        camAngleValue.remove("")
+                    
+                    camAngleCsvData = []
+                    for camAngle in camAngleValue:
+                        camAngles = camAngle.split(',')
+                        camAngleRPY = [camAngles[1], camAngles[2], camAngles[3]]
+                        camAngleCsvData.append(camAngleRPY)
+        
+                    if deviceName in ['Monitor', 'VehicleLCD', 'Laptop']:                        
+                        CamAngleCsv = pd.DataFrame(camAngleCsvData, columns = ['Roll','Pitch','Yaw'])
+                        
+                        for videoPath in videoPathList:
+                            saveName = videoPath.replace("/RGB/","/CamAngle/").replace("_rgb_","_cam_").replace(".mp4",".csv")
+                            CamAngleCsv.to_csv(saveName, index=False)
+                            print(saveName)
+                        break
+                    
+                    elif deviceName in ['Tablet', 'Smartphone']:                       
+                        CamAngleCsv = pd.DataFrame(camAngleCsvData, columns = ['Roll','Pitch','Yaw'])
+                        saveName = camAnglePath.replace(".txt",".csv")
+                        CamAngleCsv.to_csv(saveName, index=False)
+
+
+# Move .csv files under new directory
+targets = ['CamAngle','DistCam2Face','DistDisp2Face']
+move_csv(targets)
+
+# Move .mp4 files under new directory
+vidlist = glob.glob("pro*/S1/*/T1/*/RGB/*.mp4")
+vidlist.sort()
+move_vids(vidlist)
```

Comment:
would be nice if everything is put into functions, and there is 'main' functions to call them all.
But seeing this is a preprocessing script that likely to run just once (the preprocess is not called in any other file), we can do it this way.